docs/index.md

```diff
@@ -0,0 +1,98 @@
+## Overview
+This guide walk-through the process of deploying an Sample Java application in CFT.
```

Comment:
Can the site be called: `Golden path - Java`
https://github.com/hmcts/golden-path-java/blob/master/mkdocs.yml#L1

and the Home page could probably be called `Guide` or `Tutorial` for now.

Reply:
```suggestion
This guide will walk you through the process of deploying a sample Java application in CFT.
```